beacon-chain/node/config.go

```diff
@@ -117,7 +118,15 @@ func configureExecutionSetting(cliCtx *cli.Context) error {
 	if !common.IsHexAddress(ha) {
 		return fmt.Errorf("%s is not a valid fee recipient address", ha)
 	}
-	c.DefaultFeeRecipient = common.HexToAddress(ha)
+	mixedcaseAddress, err := common.NewMixedcaseAddressFromString(ha)
+	if err != nil {
+		return errors.Wrapf(err, "could not decode fee recipient %s", ha)
+	}
+	checksumAddress := common.HexToAddress(ha)
+	if !mixedcaseAddress.ValidChecksum() {
+		log.Warnf("fee recipient %s is not a valid checksum address. The checksummed address is %s and this will be used as the fee recipient. We recommend using a mixed-case address ( checksummed address ) to prevent spelling mistakes in your fee recipient Ethereum address", ha, checksumAddress.Hex())
```

Comment:
```suggestion
		log.Warnf("fee recipient %s is not a checksum Ethereum address. The checksummed address is %s and will be used as the fee recipient. We recommend using a mixed-case address (checksum) to prevent spelling mistakes in your fee recipient Ethereum address", ha, checksumAddress.Hex())
```